include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -740,10 +740,14 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
                 // getPointer() will wait for device->host transfer
                 ValueType* nativePtr = buffer.getHostBuffer().getPointer();
                 ReinterpretedType* rawPtr = reinterpret_cast<ReinterpretedType*>(nativePtr);
+#if OPENPMDAPI_VERSION_GE(0, 15, 0)
```

Comment:
**BUG** This makro was introduced with openpmd-api 0.14.5 and therefore the usage of this makro is breaking the compatibility to older version.